x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -729,6 +730,11 @@ public static RoleDescriptor kibanaSystemRoleDescriptor(String name) {
                         ".metrics-endpoint.metadata_united_default"
                     )
                     .privileges("create_index", "delete_index", "read", "index")
+                    .build(),
+                // For ILM policy for APM & Endpoint packages
+                RoleDescriptor.IndicesPrivileges.builder()
+                    .indices(".logs-endpoint.diagnostic.collection-*", "traces-apm.sampled-*")
+                    .privileges(DeleteIndexAction.NAME, RolloverAction.NAME)
```

Comment:
The `RolloverAction.NAME` privilege is already granted on lin 719 and not needed here.
Also Nit: can we move this new section under line 720 so it is closer to where related privileges are located?